Support updates (#6797)
PostHog/posthog.com: contents/docs/support-options.md

</table>
</div>

## Community Support

> There are four ways to get free support:

Reviewer: ```suggestion
There are three ways to get community support:
```

Reviewer: We should also delete the in-app option from this list, as it appears later and is confusing (as it's not community support, it's from PostHog)

> We are very grateful for folks that do this the other way around and help answer others' questions. You may just end up with some [merch](https://merch.posthog.com/) for particularly good answers :)

Reviewer: ```suggestion
We are very grateful to people that do this the other way around and help answer others' questions. You may just end up with some [merch](https://merch.posthog.com/) for particularly good answers!
```

## In-app Support

> Anybody using PostHog Cloud can raise a bug, submit feedback or otherwise ask for help via the help menu in the top right hand corner of the UI.  Your request will be routed to the correct team and we'll get back to you via email for further help if needed.

Reviewer: @MarconLP do we want to set any kind of expectation here?

Reviewer 2: We could list my focus goals here:
- high priority: 1 day response
- normal priority: 3 day response

But those would only apply to the product analytics and pipeline team tickets

Reviewer 2: If we decide to put that somewhere the pricing page would probably be the better option.

## Dedicated Slack

> Our preference for paid customers is to support them through a private Slack channel. This is because it helps teams realize they're on the same side - it's just more fun for everyone, which helps drive engagement.

Reviewer: ```suggestion
Our preference for paid customers is to support them through a private Slack channel. This helps us all get on the same side - and it's just more fun for everyone.
```
